docs/source/usage.rst

```diff
@@ -782,6 +782,24 @@ and ``OmegaConf.is_list(cfg)`` is equivalent to ``isinstance(cfg, ListConfig)``.
     >>> assert not OmegaConf.is_dict(l)
 
 
+OmegaConf.missing_keys
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Gives a set of missing keys represented in a dotlist style.
+This utility function can be used at the end of creating configuration, after merging sources and so on,
+to check for remaining mandatory fields and prompt a proper error message.
+
+.. doctest::
+
+    >>> missings = OmegaConf.missing_keys({
+    ...     "foo": {"bar": "???"},
+    ...     "missing": "???",
+    ...      "list": ["a", None, "???"]
+    ... })
+    >>> assert missings == {'list[2]', 'foo.bar', 'missing'}
+
+A plain dict, list or any valid input for `OmegaConf.create` is also acceptable - the function would raise a `ValueError` exception on wrong input.
+
```

Comment:
```suggestion
```

Since we are using `_ensure_container` now instead of raising the `ValidationError` explicitly, I don't think we should document the exact circumstances under which an error is raised (`_ensure_container` is an internal API).

Reply:
The docstring of the function says that it raises ValueError on input not representing a config - maybe that is better way to put it?

Reply:
That sounds good to me.

Reply:
👌 I've changed the phrasing